howto/bug-report/landing.md

```diff
@@ -0,0 +1,36 @@
+*Applies to: Anbox Cloud, Anbox Cloud Appliance since 1.16.0*
+
+In some cases, you might encounter issues with Anbox Cloud that you are not able
+to troubleshoot yourself, or that may be bugs in the software itself.
+
+Before applying the steps in this guide, please check the instructions for
+commonly encountered problems at [How to troubleshoot Anbox
+Cloud](https://discourse.ubuntu.com/t/how-to-troubleshoot-anbox-cloud/17837)
+first.
+
+## Build a bug report
+
+Anbox containers come preinstalled with the `anbox-bug-report` utility. On a
+running container, use `amc exec` to build the report:
+```
+amc exec <container_id> -- anbox-bug-report
+```
+
+After a few seconds, `anbox-bug-report` builds a zip archive and prints its
+absolute path.
+
+## Retrieve a bug report
+
+The generated bug report can be retrieved using an LXD client:
+```
+lxc file pull <LXD_server>:<container_id>/<log_path> <target_path>
+```
+
+Replace `<log_path>` with the given by `anbox-bug-report` and use the same `<container_id>` as before.
```

Comment:
```suggestion
Replace `<log_path>` with the path that was returned by `anbox-bug-report` and use the same `<container_id>` as before.
```

Reply:
That section was removed in favor of using a single `amc exec` command.